minvws/nl-kat-coordination
# Set default findingtype risk in model instead of in bit

### Changes

Set default findingtype risk in model instead of in bit

### Demo

No functional changes

### QA notes

Turn of kat-finding-types boefje
Create Hostname with L1
See that finding types appear
Make sure that they are "Pending"
Make sure that no reports are broken with these FindingTypes

Also check with migrations: do the same as above in main and then switch to this branch and "make" again. Rerun bits. Everything should still work

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
